src/main/java/sirius/pasta/noodle/SiriusClassAliasProvider.java

```diff
@@ -62,7 +62,6 @@ public void collectAliases(BiConsumer<String, Class<?>> consumer) {
         consumer.accept("CallContext", CallContext.class);
         consumer.accept("UserContext", UserContext.class);
         consumer.accept("WebContext", WebContext.class);
-        consumer.accept("Product", Product.class);
```

Comment:
Can this have an impact on customer templates? (especially S2)

Reply:
Shouldn't be too much of an issue: We probably need to migrate a few templates anyway due to the tagiatelle changes, and we will detect any compilation errors using a job